recipes/reaktoro/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+echo CC=$CC
+echo CXX=$CXX
+which $CXX
+$CXX --version
+mkdir build
+cd build
+cmake -G Ninja \
```

Comment:
Here please use the CMAKE_GENERATOR

Reply:
Why?

Reply:
Just to avoid to use ninja in unix and use make. Because in that way you can remove one more dependency in your build for unix platforms.

Reply:
@marcelotrevisani, we're not building on Linux yet (because it will need GCC 7), I can remove `build.sh` for now ;)